kimchi_description/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kimchi_description</name>
+  <version>0.2.0</version>
+  <description>The kimchi_description package</description>
+  <author email="franco.c@ekumenlabs.com">Franco Cipollone</author>
+  <maintainer email="franco.c@ekumenlabs.com">Franco Cipollone</maintainer>
+  <license file="LICENSE">BSD Clause 3</license>
```

Comment:
Our repo says it uses Apache license. (I know it also has to be fixed in other pks)

Reply:
Dismiss this comment